lib/App/GitHooks.pm

```diff
@@ -452,6 +452,8 @@ sub run
 		croak "Invalid hook name $name"
 			if scalar( grep { $_ eq $name } @$HOOK_NAMES ) == 0;
 
+		return $HOOK_EXIT_SUCCESS if _should_skip( $name );
```

Comment:
We may want to print a warning on STDERR here, to indicate that hooks were intentionally skipped. This will be a good reminder in case someone "inadvertently" adds this to their `.bash_profile`.